sample/DateTimeTextParser/DateTimeParser.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="superpower" Version="1.0.2" />
```

Comment:
Could this be a project reference? (more along the lines of the `IntCalc` csproj)

Reply:
Nice spotting 👍